docs/learn/learn-agile-coretime.md

```diff
@@ -1,15 +1,30 @@
 ---
 id: learn-agile-coretime
-title: Introduction to Agile Coretime
-sidebar_label: Agile Coretime Intro
-description: Introduction to Agile Coretime and its terminology
-keywords: [coretime, blockspace, parachain, on-demand, cores]
+title: Multi-Threading
+sidebar_label: Multi-Threading
+description: How the Polkadot Cloud achieves multi-threading to improve efficiency.
+keywords: [coretime, blockspace, parachain, on-demand, cores, multi-threading]
 slug: ../learn-agile-coretime
 ---
 
-Agile Coretime enables efficient utilization of Polkadot network resources and provides economic
-flexibility for builders, generalizing Polkadot beyond what was initially proposed and envisioned in
-its [whitepaper](https://polkadot.network/whitepaper/).
+import DocCardList from '@theme/DocCardList';
+
+[Multi-threading](<https://en.wikipedia.org/wiki/Multithreading_(computer_architecture)>) is a
+programming model where multiple threads (smaller sequences of programmed instructions) are created
+within a single process to perform multiple tasks at once. Multi-threading is commonly used to
+improve the performance of applications by executing different parts of a program concurrently.
+[Concurrency](<https://en.wikipedia.org/wiki/Concurrency_(computer_science)>) does not imply
```

Comment:
> flexible options to access blockspace, with access to bulk coretime or on-demand coretime

this sounds less like a technical feature, and more about sales.

when you compare to cloud platforms, your ability to purchase resources on-demand, or in bulk, or whatever seems like the comparison.

Reply:
hey @DrW3RK and @shawntabrizi yes I was thinking about interlacing when doing this PR. I agree scheduling is more appropriate for agile coretime and then we can explain that scheduling enables multi-threading. I will add this feedback to the PR.

Reply:
Thanks. Here are some chains that use multi-threading to improve performance, which are not available on the Polkadot tech stack yet. It would be great to see these features added on the collator side.

https://aptos.dev/en/network/blockchain/execution
https://blog.sui.io/parallelization-explained/